.gitignore

```diff
@@ -3,3 +3,9 @@ Dance Together/Assets/Private/
 Dance Together/Assets/Demigiant/
 *.wav
 *.wav.meta
+Dance Together/Assets/Scripts/Controllers/GameManager.cs
```

Comment:
Why is this being ignored?

Reply:
My appologies. That was legacy ignore stuff that i didnt intend on commiting.

Reply:
I figured. I merged and removed them :)